.docker/btsieve/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM rust:1.37-buster as builder
+
+WORKDIR /src
+COPY ./btsieve ./btsieve
+COPY ./internal ./internal
+COPY ./cnd ./cnd
+COPY ./libp2p-comit ./libp2p-comit
+COPY ./blockchain_contracts ./blockchain_contracts
+COPY ./Cargo.toml ./Cargo.toml
+COPY ./Cargo.lock ./Cargo.lock
+
+RUN cargo check
```

Comment:
I understand you want to fail fast but considering how slow is docker and the fact that this would only be done for a release, I think it can be skipped

Reply:
Can potentially be skipped yes but it also doesn't include _that_ much overhead since most time is spent downloading dependencies :(